webpack.config.prod.js

```diff
@@ -0,0 +1,50 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+    entry: {
+        helloWorldPage: 'pages/hello-world/helloWorld.jsx'
+    },
+
+    output: {
+        path: path.resolve(__dirname, 'public'),
+        chunkFilename: 'webpack/[chunkhash].js',
+        filename: "javascripts/[name].js",
+        publicPath: '/assets/'
+    },
+
+    resolve: {
+        alias: {
+            'react': 'preact-compat',
+            'react-dom': 'preact-compat'
+        },
+        modules: [
+            path.resolve(__dirname, "assets"),
+            path.resolve(__dirname, "node_modules")
+        ],
+        extensions: [".js"]
```

Comment:
Good catch `adcock-lint`, we are not linting the files of the root. I will add them to the lint process. Thanks